libs/ledgerjs/packages/react-native-hw-transport-ble/src/BleTransport.ts

```diff
@@ -633,8 +771,13 @@ export default class BleTransport extends Transport {
         mtu = await firstValueFrom(
           merge(
             this.notifyObservable.pipe(
-              tap(maybeError => {
-                if (maybeError instanceof Error) throw maybeError;
+              map(maybeError => {
```

Comment:
the point of replacing the `tap` by a `map` here is to guarantee that the resulting observable (created by `pipe`) errors ? if you just throw an error in `tap` it doesn't affect the overall observable created by pipe ?

Reply:
`tap` is supposed to be for "side effect" like changing an outside value or logging. But looking at ex [in the doc](https://rxjs.dev/api/operators/tap) they show an ex where they throw an error from `tap` ahah. So i guess it was fine and i was wrong to change it. I can put it back if you think it's clearer ? I just feel that `map` indicates that something is going to happen to the streamed value, while `tap` does not touch it

And to answer your 2nd question: 
>Any error that occurs -- synchronously -- in a handler provided to `tap` will be emitted as an error from the returned observable.

Reply:
nah no need to change it back 👍